packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -792,7 +792,7 @@ curl -v -X POST \
 
 A `429 Too Many Requests` status code may be returned if you attempt to resend an SMS challenge (OTP) within the same time window.
 
-> **Notes:** The current rate limit is one SMS challenge per device every 30 seconds.<br><br> Okta round-robins between SMS providers with every resend request to help ensure delivery of SMS OTP across different carriers.
+> **Notes:** The current [rate limit](/docs/reference/rate-limits/) is one SMS challenge per device every 30 seconds.<br><br> Okta round-robins between SMS providers with every resend request to help ensure delivery of SMS OTP across different carriers.
```

Comment:
This should say the rate limit is per phone number as Eric pointed out this is not per device.